dss/storage/checkout/cache_flow.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+from dss.storage.hcablobstore import FileMetadata
+
+"""
+These functions assist with the caching process, the lifecycle policies are set to delete files within
+the checkout buckets;
+For AWS object tagging is used to mark files for deletion: TagSet=[{uncached:True}]
+For GCP object classes are used to indicate what is to be cached: STANDARD_STORAGE (MULTI-REGIONAL) are cached
+See MetaData Caching RFC for more information (Google Docs)
+"""
+
+
+def check_dss_bucket(dst_bucket: str):
+    return dst_bucket in (os.environ['DSS_GS_CHECKOUT_BUCKET_TEST'], os.environ['DSS_S3_CHECKOUT_BUCKET_TEST'],
+                          os.environ['DSS_S3_CHECKOUT_BUCKET'], os.environ['DSS_GS_CHECKOUT_BUCKET'])
+
+
+def get_cache_criteria():
+    """Fetch criteria to determine if files should be cached."""
+    if os.getenv("CHECKOUT_CACHE_CRITERIA"):
+        return json.loads(os.getenv('CHECKOUT_CACHE_CRITERIA'))
+
+    local_cache_criteria_path = f'{os.environ["DSS_HOME"]}/checkout_cache_criteria.json'
```

Comment:
If `CHECKOUT_CACHE_CRITERIA` is set in `environment` then we don't need the fallback file.